Go update error

During go update of bettercap:
`go/src/github.com/bettercap/bettercap/modules/gps.go:133:15: info.GetSentence undefined (type nmea.Message has no field or method GetSentence)`

### Environment

`Darwin Macbook.local 16.7.0 Darwin Kernel Version 16.7.0: root:xnu-3789.73.8~1/RELEASE_X86_64 x86_64`
`go version go1.10 darwin/amd64`
### Steps to Reproduce

Run
`go get -u -v github.com/bettercap/bettercap`
or
`go get -u github.com/bettercap/bettercap`

**Expected behavior:** *Updated bettercap*

**Actual behavior:** *Nothing happened*

-- 

**♥ ANY INCOMPLETE REPORT WILL BE CLOSED RIGHT AWAY ♥**

same issue when I try to install from stratch
**During installation of bettercap:**  github.com/bettercap/bettercap/modules
src/github.com/bettercap/bettercap/modules/gps.go:133:15: info.GetSentence undefined (type nmea.Message has no field or method GetSentence)
**Enviroment:** Linux  4.4.0-116-generic #140-Ubuntu SMP Mon Feb 12 21:23:04 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux
go version go1.10 linux/amd64

Same here:

root@kali-desktop:~/go/bin# go get -u github.com/bettercap/bettercap
# github.com/bettercap/bettercap/modules
../src/github.com/bettercap/bettercap/modules/gps.go:133:15: info.GetSentence undefined (type nmea.Message has no field or method GetSentence)
root@kali-desktop:~/go/bin# cat /etc/*release
DISTRIB_ID=Kali
DISTRIB_RELEASE=kali-rolling
DISTRIB_CODENAME=kali-rolling
DISTRIB_DESCRIPTION="Kali GNU/Linux Rolling"
PRETTY_NAME="Kali GNU/Linux Rolling"
NAME="Kali GNU/Linux"
ID=kali
VERSION="2018.1"
VERSION_ID="2018.1"
ID_LIKE=debian
ANSI_COLOR="1;31"
HOME_URL="http://www.kali.org/"
SUPPORT_URL="http://forums.kali.org/"
BUG_REPORT_URL="http://bugs.kali.org/"
root@kali-desktop:~/go/bin# go version
go version go1.9.2 linux/amd64
root@kali-desktop:~/go/bin# uname -a
Linux kali-desktop 4.14.0-kali3-amd64 #1 SMP Debian 4.14.12-2kali1 (2018-01-08) x86_64 GNU/Linux

--- a/modules/gps.go
+++ b/modules/gps.go
@@ -130,7 +130,7 @@ func (gps *GPS) Start() error {
 		for gps.Running() {
 			if line, err := gps.readLine(); err == nil {
 				if info, err := nmea.Parse(line); err == nil {
-					s := info.GetSentence()
+					s := info.Sentence()
 					// http://aprs.gids.nl/nmea/#gga
 					if s.Type == "GNGGA" {
 						gps.Session.GPS = info.(nmea.GNGGA)

